client/src/components/WorkspaceDialog.vue

```diff
@@ -75,7 +75,7 @@ export default Vue.extend({
   methods: {
     async create() {
       if (this.newWorkspace) {
-        const response = await api().post(`/workspaces/${this.newWorkspace}`);
+        const response = api.createWorkspace(this.newWorkspace);
```

Comment:
Should we await here? Looks like this function returns a promise

Reply:
Good catch. If I had explicitly typed the `response` variable, typescript would have caught this for me :open_mouth:

Fixed in 1a82a7d76c7ccc058e42a1dbd85ce7c4ff1f639d.